src/components/stories/stories.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import storiesDefaultImage from './assets/stories-default.jpg';
+
+import styles from './stories-styles.scss';
+
+class Stories extends PureComponent {
+  // eslint-disable-line react/prefer-stateless-function
+  render() {
+    const { stories, handleClickAnalytics, storiesTitle, theme } = this.props;
```

Comment:
I'd rename the name of the method for something more generic, not necessarily needs to be used only for analytics.